packages/core/src/modules/basic-messages/messages/BasicMessage.ts

```diff
@@ -6,6 +6,8 @@ import { IsValidMessageType, parseMessageType } from '../../../utils/messageType
 import { DateParser } from '../../../utils/transformers'
 
 export class BasicMessage extends AgentMessage {
+  public readonly protocolUsesLegacyDidSovPrefix = true
```

Comment:
Maybe `protocolUsedDidSovPrefix`?

`canUse` is a bit vague to me as every protocol _could_ use it.

Reply:
How about `protocolAllowsDidSovPrefix`? If you prefer `protocolUsedDidSovPrefix` I am okay with that. But it kind of sounds to me like it can not anymore but it used to.

Reply:
protocolAllowsDidSovPrefix Sounds good to me

Reply:
Maybe just allowDidSovPrefix?

Reply:
That means the message class allows the prefix or not